beebop/filestore.py

```diff
@@ -227,11 +239,42 @@ def tmp(self, p_hash) -> str:
         os.makedirs(tmp_path, exist_ok=True)
         return str(tmp_path)
 
+    def output_tmp(self, p_hash) -> str:
+        """
+        Generates the path to the full assign output folder.
```

Comment:
```suggestion
        Generates the path to the full assign output folder when using full db.
```
..this is just used for full db assign isn't it?